### PR TITLE
85 layout refactoring

### LINKED ISSUE #51: Validate btn style
Add style to the validate input.

Also, would be nice to display it:

- on the whole page (even when scrolling),
- only when there are some changes to commit.

### LINKED ISSUE #85: Layout refactoring
The way that are handle the styles for the layout (`Sidebar`, padding around the main content, ...) need to be refactored to be more customizable.

### PR DESCRIPTION
**Pull request by PaulFarault**
#### Which issue(s) this PR fixes

Fixes #85 
Fixes #51 

#### Agreements

- [X] I hereby declare this contribution to be licensed under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0).
- [X] I hereby agree to grant [TOSIT](https://www.tosit.io/) a copyright license to use my contributions.

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -36,3 +36,6 @@ yarn-error.log*
 # typescript
 *.tsbuildinfo
 next-env.d.ts
+
+# docker
+/docker
```
Comment (gboutry) [inline .prettierignore:41]:
```suggestion
/docker

```
missing new linea

File: src/app/dashboard/layout.tsx [hunk 2]
```diff
@@ -18,15 +23,26 @@ export default function DashboardLayout({ children }) {
...
-        <Sidebar className="basis-60 flex-grow" />
-        <main className="basis-0 grow-[999] overflow-auto">
-          <div className="mx-auto max-w-7xl px-4 sm:px-6 lg:px-8">
-            <div className="mx-auto max-w-5xl py-10">{children}</div>
+      <Sidebar sideWidth="13rem" space="0">
+        <aside className="h-screen overflow-auto bg-gray-900 flex flex-col">
+          <div className="mb-3 py-3 self-center">
+            <p className="flex">
+              <Image
+                src="/TDP_LOGO_INVERSE_notext.png"
```
Comment (gboutry) [inline src/app/dashboard/layout.tsx:31]: Are you using git lfs to store the image, or directly storing in the repo ?

File: src/hooks/useDeployLogInfos.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import { useEffect, useState } from 'react'
+import { useTdpClient } from 'src/contexts'
+
+import type { DeploymentLogWithOperations } from '@/client-sdk'
+
+export function useDeployLogInfos(deployLogId: number) {
```
Comment (gboutry) [inline src/hooks/useDeployLogInfos.ts:6]: This doesn't look like it's used in the project, why are you adding this in `layout-refactoring' ?

**Review by gboutry — CHANGES_REQUESTED** [review]

File: src/app/dashboard/layout.tsx (hunk 2 above)
Comment (PaulFarault) [reply src/app/dashboard/layout.tsx:31]: No, it doesn't seem to work with this repo:

```
This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access.
```

**Review by gboutry — CHANGES_REQUESTED** [review]
Static export does not work, I had to add `next.config.js`:

```js
const nextConfig = {
    images: {
        unoptimized: true
    }
}

module.exports  = nextConfig;
```

**Review by gboutry — APPROVED** [review]

**Pull request merged**